macros/sql/groupby.sql

```diff
@@ -1,9 +1,8 @@
-{% macro group_by(n) %}
+{%- macro group_by(n) -%}
 
   group by
-    {% for i in range(1, n + 1) %}
-      {{ i }}
-      {% if not loop.last %} , {% endif %}
-   {% endfor %}
+    {% for i in range(1, n + 1) -%}
+      {{ i }}{{ ',' if not loop.last }}   
+   {%- endfor -%}
```

Comment:
Actually - one minor nit - can we make this read:
```
{%- macro group_by(n) -%}
  group by {% for i in range(1, n + 1) -%}
      {{ i }}{{ ',' if not loop.last }}   
   {%- endfor -%}
{%- endmacro -%}
```
The difference is that the SQL renders out as
```
select
  1 as id,
  2 as name
group by 1,2
```
instead of
```
select
  1 as id,
  2 as name
group by
  1,2
```

Reply:
done!